app/components/DataInput/index.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
```

Comment:
I like this thought, but I'd prefer to do it as a separate PR that covers all components